career/working-and-collaborating-remotely.md

```diff
@@ -0,0 +1,172 @@
+
+### Independent Practice
+With a colleague or fellow apprentice, do a video conference call, and help each other determine if their sound levels and lighting are acceptable, and the background appears professional.
+
+### Challenge
+In this challenge, you will use a professional online video conferencing platform and a kanban style board to solve a handful of coding challenges
+  1. Form a team with 1 or 2 fellow apprentices or colleagues outside of Techtonica
+  1. Assign one person to open an online video conferencing account on [Zoom](https://zoom.us/pricing) and start a session with the other participants
+  1. Have another person open a Google Sheet to make a kanban board. See this [post-it kanban board](https://lifehacker.com/productivity-101-how-to-use-personal-kanban-to-visuali-1687948640) for an example. Add 3 columns to the board titled "Todo", "Doing", "Done", and share it with the group so all can edit
+  1. Then have everyone navigate to [Code Wars](https://www.codewars.com/) and pick out a handful of kata challenges in JavaScript (beginners should screen for Kyu (level) 7 or 8 challenges). Click on the ones that seem interesting and copy and paste the links into the "Todo" column of the kanban board
+  1. Once you have a list of 6-10 challenges, pick out one challenge for each person, and move those into the "Doing" column on the kanban board
+  1. Start working on the challenges
+  1. When the first person has completed a challenge, have them move the link into the "Done" column, and then pair program with one of the other people that are still working on their challenge until completed
+  1. If the team has a total of two people, start another round picking two new challenges from the "Todo" column. Otherwise, if there is a third person, have the two that just wrapped up the second code challenge together team up with the third person to complete the round. Then again, pick out 3 new challenges, and repeat
+
+See how many challenges you can get through, and save the sheet in case you have time in the future to solve more challenges. Seeing the list grow will build your confidence!
+
+### Check for Understanding
+Go onto Youtube or search for blog posts about the “pros and cons” of remote work, the etiquette of communicating online, or professional growth for remote workers. Write a short summary of what you found to be a surprise or misconception, or on the other hand, solidified your beliefs about the subject.
+
+### Supplemental Materials
```

Comment:
DHH's blog might be good to include. IIRC his book remote was made up mostly of blog content.

Related, here are some cool setups:
https://m.signalvnoise.com/remote-working-the-home-office-desks-of-basecamp/

Reply:
Added the signal to noise blog if that's what you meant, I wasn't sure. Also included a list of blogs that deal with remote work